driver/scsi_operation.c

```diff
@@ -229,6 +229,7 @@ UCHAR
 WnbdSetModeSense(_In_ PVOID Data,
                  _In_ PCDB Cdb,
                  _In_ ULONG MaxLength,
+                 _In_ BOOLEAN ReadOnly,
```

Comment:
It might be better to pass `PSCSI_DEVICE_INFORMATION` here

Reply:
Well, we only need one flag. It should be fine for now.